scripts/30mini-iso-menu

```diff
@@ -26,16 +26,14 @@ iso_chooser_step1() {
     # use the requested ISO size to figure out if we have the memory or not,
     # and kexec to reserve that memory
 
-    echo "###########################################"
-    echo "iso_chooser_step1"
-    echo "###########################################"
-
+    chvt 2  # the chvts work around messages bleeding into the agetty
```

Comment:
Do we want this to work at all over serial? I don't really know the implications of that question but this isn't going to work i think...

Reply:
I expect it won't, and want to investigate in a future PR.